src/assets/scss/app.scss

```diff
@@ -0,0 +1,209 @@
+@import 'reset.css';
+@import "variables";
+@import "button";
+@import "modal";
+
+body {
+    background: #555555;
+    color: #d4d4d4;
+    font-size: 18px;
+}
+
+#app {
+    width: $width_app;
+    margin: 0 auto;
+    position: relative;
+
+    .header {
+        width: 100%;
+        height: 395px;
+        background: $color2;
+        margin: 53px 0 10px 0;
+
+        background-image: url('../img/background_header.png');
+        background-position: bottom;
+        background-repeat: no-repeat;
+        background-size: cover;
+
+        .blur {
+            background: rgb(0 0 0 / 65%);
+            backdrop-filter: blur(4px);
+            height: 395px;
+            width: 100%;
+        }
+        
+        .top {
+            display: flex;
+        
+            .logo {
+                padding: 30px 0 0 60px;
+            }
+            & > .btn {
+                margin: 20px 50px 0 0;
+                position: absolute;
+                right: 0;
+                color: $color1;
+            }
+        }
+        
+        .search {
+            width: 82.33%;
+            margin: 55px auto 0 auto;
+
+            &>.title {
+                font-size: 50px;
+                padding: 3px;
+            }
+            &>.wrap {
+                display: flex;
+                margin-top: 30px;
+
+                input {
+                    width: 710px;
+                    height: 50px;
+                    margin: 5px;
+                    padding-left: 10px;
+                    background-color: $color3;
+                    color: $color4;
+                    border: 0;
+                    box-shadow: 0 0 8px 3px $color3;
+                }
+            }
+        }
+        .movie-details {
+            .blur {
+                background: rgb(0 0 0 / 88%);
+            }
+
+            .content {
+                display: flex;
+                margin: 30px 60px 0 60px;
+
+                img {
+                    margin-right: 60px;
+                }
+                h1 {
+                    font-size: 50px;
+                    margin-bottom: 30px;
+                    .contentRating {
+                        margin-left: 30px;
+                        border: 1px solid $color5;
+                        border-radius: 50%;
+                        width: 50px;
+                        height: 50px;
+                        display: inline-block;
```

Comment:
You don't have `R` in center of the span. So maybe you can use something like this?
```css
display: inline-flex;
justify-content: center;
```